backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/query/impl/ElasticsearchSearchResultExtractorImpl.java

```diff
@@ -20,21 +20,25 @@
 import org.hibernate.search.engine.search.SearchResult;
 import org.hibernate.search.engine.search.query.spi.LoadingResult;
 import org.hibernate.search.engine.search.query.spi.ProjectionHitMapper;
+import org.hibernate.search.util.AssertionFailure;
 
 import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 
 public class ElasticsearchSearchResultExtractorImpl<T> implements ElasticsearchSearchResultExtractor<T> {
 
-	private static final JsonObjectAccessor HITS_ACCESSOR =
-			JsonAccessor.root().property( "hits" ).asObject();
+	private static final JsonObjectAccessor HITS_ACCESSOR = JsonAccessor.root().property( "hits" ).asObject();
 
-	private static final JsonAccessor<JsonArray> HITS_HITS_ACCESSOR =
-			HITS_ACCESSOR.property( "hits" ).asArray();
+	private static final JsonAccessor<JsonArray> HITS_HITS_ACCESSOR = HITS_ACCESSOR.property( "hits" ).asArray();
 
-	private static final JsonAccessor<Long> HITS_TOTAL_ACCESSOR =
-			HITS_ACCESSOR.property( "total" ).asLong();
+	private static final JsonAccessor<Long> HITS_TOTAL_ACCESSOR = HITS_ACCESSOR.property( "total" ).asLong();
+
+	private static final JsonAccessor<Integer> TOOK_ACCESSOR = JsonAccessor.root().property( "took" ).asInteger();
+
+	private static final JsonAccessor<Boolean> TIMED_OUT_ACCESSOR = JsonAccessor.root().property( "timed_out" ).asBoolean();
+
+	private static final JsonAccessor<String> SCROLL_ID_ACCESSOR = JsonAccessor.root().property( "_scroll_id" ).asString();
```

Comment:
You can remove the added accessors, they are not useful. See also my comment on `SearchResult.java`